public/components/custom_panels/redux/panel_slice.ts

```diff
@@ -141,6 +141,13 @@ export const uuidRx = /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a
 
 export const isUuid = (id) => !!id.match(uuidRx);
 
+export const doesNameExist = (newCustomPanelName: string) => async() => {
+  const panels = await fetchCustomPanels();
+  if((panels.some((i: { title: string; }) => i.title === newCustomPanelName))){
```

Comment:
nit. didn't run lint? missing space after `if`. Also probably `reutrn panels.some((i: { title: string; }) => i.title === newCustomPanelName)`?

and what happens if fetch call failed?